packages/moonstone/styles/internal/fonts.less

```diff
@@ -1,8 +1,8 @@
 // ----- SYSTEM FONT NAMES ------
-@font-system-src-moonstone-condensed-300:         local("LGSmartUICondTest-Light"), local("Miso"), url("../../fonts/Miso/Miso-Light.ttf") format("truetype");
-@font-system-src-moonstone-condensed-400:         local("LGSmartUICondTest-Regular"), local("Miso"), url("../../fonts/Miso/Miso-Regular.ttf") format("truetype");
-@font-system-src-moonstone-condensed-600:         local("LGSmartUICondTest-SemiBold"), local("Miso"), url("../../fonts/Miso/Miso-Bold.ttf") format("truetype");
-@font-system-src-moonstone-condensed-700:         local("LGSmartUICondTest-Bold"), local("Miso"), url("../../fonts/Miso/Miso-Bold.ttf") format("truetype");
+@font-system-src-moonstone-condensed-300:  local("LGSmartUICond-Light"), local("Miso"), url("../../fonts/Miso/Miso-Light.ttf") format("truetype");
```

Comment:
I think We should use font family instead of PS name in this PR.

Reply:
You know... I think you're right. The TV appears to not leverage the PostScript names, but does seem to be able to access the full-name, rather than just the family-name. I'll make a PR to update our fonts file to refer to the full-names instead. Thanks.